its/src/org/sonarlint/eclipse/its/reddeer/views/RuleDescriptionView.java

```diff
@@ -50,7 +50,7 @@ public Label getRuleSeverity() {
   }
 
   public Label getRuleKey() {
-    return new DefaultLabel(getCTabItem(), 5);
+    return new DefaultLabel(getCTabItem(), 8);
```

Comment:
What does this 8 represent? Maybe you could extract it as a constant and give it a meaningful name?

Reply:
Yeah, this is on my agenda to clarify that magic number when I'll be implementing all the ITs / UTs. It is currently switched to 8 because we have 8 labels for the Rule Description view header (where clean code attribute / impacts / rule key live). 5 will be for the IT with the connection to an old SQ with the legacy header shown.